src/Courses/CourseInterfaces.tsx

```diff
@@ -129,6 +129,7 @@ export class StudentGrade {
     bestScore: number = 0;
     numAttempts: number = 0;
     locked: boolean = false;
+    currentProblemState?: JSON;
```

Comment:
Interesting, I was unaware of a json data type, seems like something the linter wouldn't be happy with since it seems like another way to say `any`